src/main/io/beeper.c

```diff
@@ -197,12 +193,12 @@ typedef struct beeperTableEntry_s {
     uint8_t mode;
     uint8_t priority; // 0 = Highest
     const uint8_t *sequence;
-#ifdef BEEPER_NAMES
+#ifdef USE_BEEPER_NAMES
```

Comment:
This change is intended for this PR?

Reply:
Hmmm... I found it while I was looking for occurrences of `FLASH_SIZE`, and fixed it along side. Do you think it's better to file it as a separate PR?

Reply:
I suppose it's better, but I understand the cause 😉 so to me is ok

Reply:
I'm updating the PR not to include the BEEPER_NAMES change.